code/modules/recycling/disposal.dm

```diff
@@ -143,8 +143,12 @@
 	user.drop_item()
 	if(I)
 		if(istype(I, /obj/item/weapon/holder/micro))
-			log_and_message_admins("placed [I.name]  inside \the [src]", user)
-		I.forceMove(src)
+			log_and_message_admins("placed [I.name] inside \the [src]", user)
+			var/obj/item/weapon/holder/H = I
+			H.held_mob.forceMove(src)
+		else
+			I.forceMove(src)
+			explosion()
```

Comment:
No

Reply:
Legitimate mistake that I mustve forgotten to delete when trying to find the parameters for explosion, my bad